eq-author-api/schema/typeDefs.js

```diff
@@ -575,7 +575,7 @@ type Mutation {
   createOption(input: CreateOptionInput!): Option
   createMutuallyExclusiveOption(input: CreateMutuallyExclusiveOptionInput!): Option
   updateOption(input: UpdateOptionInput!): Option
-  deleteOption(input: DeleteOptionInput!): Option
+  deleteOption(input: DeleteOptionInput!): MultipleChoiceAnswer
```

Comment:
Now that the API has changed, the resolver needs to be updated to return the answer instead of the option. https://github.com/ONSdigital/eq-author-app/blob/fc7e8ea7bcc8b18751228e905c2d9a147fa2b503/eq-author-api/schema/resolvers/base.js#L347

Reply:
Now done.